File: google_earth_engine/utils/assets_delete.py
# Initial imports.
import ee
import json
import pathlib
from typing import Dict, Any


# Main parameters.
key_path: str = "/geosatlearn_app/gee_key_alternative_data.json"
folder_path: str = "projects/hblhug67j2mi/assets/bayer-api-tests"


# Ancillary function to initialize the Earth Engine library.
def gee_quick_init() -> None:
    gee_key_path: pathlib.Path = pathlib.Path(key_path)
    with open(gee_key_path) as gee_key_file:
        ee.Initialize(
            ee.ServiceAccountCredentials(
                json.load(gee_key_file)["client_email"], str(gee_key_path)
            ),
            opt_url="https://earthengine-highvolume.googleapis.com",
        )


# Initialize the Earth Engine library.
gee_quick_init()

# List all assets.
assets: Dict[str, Any] = ee.data.listAssets({"parent": folder_path})

# Delete all assets.
for asset in assets.get("assets", []):
    asset_id: str = asset["id"]
    print(f">>> Deleting asset: {asset_id}")
    ee.data.deleteAsset(asset_id)

print(">>> All assets deleted.")

File: google_earth_engine/utils/assets_delete.js
// Caminho da pasta no GEE (exemplo: 'users/seu_usuario/sua_pasta')
var folderPath = "projects/hblhug67j2mi/assets/bayer-api-load-tests/";

// Função para deletar todos os assets em uma pasta.
var deleteAssetsInFolder = function(folder) {
  
  var assetsList = ee.data.listAssets(folder);
  if (assetsList.assets && assetsList.assets.length > 0) {
    
    assetsList.assets.forEach(function(asset) {
      print("Deleting asset:", asset.id);
      ee.data.deleteAsset(asset.id);
    });
    
    print("All assets deleted.");
  
    
  } else {
    print("No assets found in folder:", folder);
  }

  
};

// Chame a função passando o caminho da sua pasta.
deleteAssetsInFolder(folderPath);